planner/core/physical_plan_test.go

```diff
@@ -445,6 +445,74 @@ func (s *testPlanSuite) TestAggEliminator(c *C) {
 	}
 }
 
+func (s *testPlanSuite) TestEliminateMaxOneRow(c *C) {
```

Comment:
We can merge two test function into one.

Reply:
OK，done